lesson 0/README.md

```diff
@@ -1,18 +1,31 @@
 # Lesson 0 - Getting prepared
 
-Welcome to [insert class name here]! We're excited to be with you this semester, and in order to have a smoother process there are a few things that we'd like you to do before coming in to class on September 1st.
+Welcome to MobileProto! We're excited to be with you this semester, and in order to have a smoother process there are a few things that we'd like you to do before coming in to class on September 1st.
 
-## 1. Download Android Studio
+## Install Ubuntu 14.04
 
-If you do this before class we'll have a lot more time to help you out and get started! Please download Android Studio 2.x [link here].
+This is the recommended OS for this course. If you don't already have Ubuntu on your computer either via dual-boot or a VM, we strongly recommend you to install it. You can use Windows or OSX, but if you have OS-specific issues it will be more difficult for the teaching team to help you.
+
+## 1. Download and Install Android Studio
+
+If you do this before class we'll have a lot more time to help you out and get started! Please download [Android Studio 2.1](https://developer.android.com/studio/install.html).
```

Comment:
as a side note, the reason I left it 2.x is because this version will probably update over the summer. We just need to remember to change and update this in August